src/pybind/mgr/requirements-mgr.lock.txt

```diff
@@ -0,0 +1,243 @@
+#
+# This file is autogenerated by pip-compile with Python 3.10
+# by the following command:
+#
+#    pip-compile --allow-unsafe --generate-hashes --output-file=requirements-mgr.lock.txt --resolver=backtracking requirements-mgr.txt
```

Comment:
I'm not familiar with dependabot in this case tbh. I simply named it lock file because it looks like it and is generated from a simpler requirements file.

Reply:
What about using the [file names suggested by the project](https://github.com/jazzband/pip-tools#requirements-from-requirementsin)?